### ISSUE TITLE
Missning sequence for some cameras

### ISSUE THREAD

**Issue by anilesec** [facebookresearch/ContactPose#15]
Hi @samarth-robo 

I downloaded the dataset but for some subjects with certain objects sequences is missing for one of the 3 cameras.
For ex: full3_handoff, scissors, kinect2_left case the color images dir do not exist. Is this intentional? or am I missing something in the download?

Thanks in advance for the response±

**Comment by anilesec** [comment]
Another case is for full2_handoff, apple: annotations are available but images are not. is this expected? or is there any problem with my download?

**Comment by samarth-robo** [comment]
@anilesec I am not sure why annotations are present for that camera, but images are not. Can you see if the `download_data.py` script has a Dropbox link for that camera's colour images?

**Comment by anilesec** [comment]
Thanks, @samarth-robo for the response. I checked the data/urls.json and for full2_handoff-apple sequence colour images, the link is not available.

**Comment by samarth-robo** [comment]
Yes that is rare but possible. In `annotations.json` for full2_handoff-apple, look at the `cameras` section. Do they have `valid: True`?

**Comment by anilesec** [comment]
Yes, it has all three `valid: True` for all 3 cameras

**Comment by ywang1121** [comment]
> Hi [@samarth-robo](https://github.com/samarth-robo)
> 
> I downloaded the dataset but for some subjects with certain objects sequences is missing for one of the 3 cameras. For ex: full3_handoff, scissors, kinect2_left case the color images dir do not exist. Is this intentional? or am I missing something in the download?
> 
> Thanks in advance for the response±

Hello, the dataset link provided by the author is no longer valid now. I would like to download this dataset. Do you have this dataset that you could share with me? If so, I would be extremely grateful!
